### PR TITLE
Manual updates 20221004 gps only

### PR DESCRIPTION
**Pull request by moljac**
### Does this change any of the generated binding API's?

- `com.google.android.gms:playservices-ads` - 21.1.0 -> 21.2.0
- `com.google.android.gms:playservices-ads-base` - 21.1.0 -> 21.2.0
- `com.google.android.gms:playservices-ads-lite` - 21.1.0 -> 21.2.0
- `com.google.android.gms:playservices-analytics` - 18.0.1 -> 18.0.2
- `com.google.android.gms:playservices-analytics-impl` - 18.0.1 -> 18.0.2
- `com.google.android.gms:playservices-auth` - 20.2.0 -> 20.3.0
- `com.google.android.gms:playservices-auth-base` - 18.0.3 -> 18.0.5
- `com.google.android.gms:playservices-games` - 22.0.1 -> 23.0.0
- `com.google.android.gms:playservices-games-v2` -  -> 17.0.0
- `com.google.android.gms:playservices-measurement` - 21.1.0 -> 21.1.1
- `com.google.android.gms:playservices-measurement-api` - 21.1.0 -> 21.1.1
- `com.google.android.gms:playservices-measurement-base` - 21.1.0 -> 21.1.1
- `com.google.android.gms:playservices-measurement-impl` - 21.1.0 -> 21.1.1
- `com.google.android.gms:playservices-measurement-sdk` - 21.1.0 -> 21.1.1
- `com.google.android.gms:playservices-measurement-sdk-api` - 21.1.0 -> 21.1.1
- `com.google.android.gms:playservices-mlkittextre-cognition` - 18.0.1 -> 18.0.2
- `com.google.android.gms:playservices-mlkittextre-cognition-common` - 17.1.0 -> 18.0.0
- `com.google.android.gms:playservices-tagmanager` - 18.0.1 -> 18.0.2
- `com.google.android.gms:playservices-tagmanager-api` - 18.0.1 -> 18.0.2
- `com.google.android.gms:playservices-tagmanager-v4-impl` - 18.0.1 -> 18.0.2
- `com.google.android.gms:playservices-tflite-gpu` -  -> 16.0.0
- `com.google.android.gms:playservices-tflite-impl` -  -> 16.0.0
- `com.google.android.gms:playservices-tflite-java` -  -> 16.0.0
- `com.google.android.gms:playservices-tflite-support` -  -> 16.0.0
- `com.google.android.gms:playservices-wearable` - 17.1.0 -> 18.0.0
- `org.tensorflow:tensorflow-lite` - 2.6.0 -> 2.10.0
- `org.tensorflow:tensorflow-lite-api` -  -> 2.10.0
- `org.tensorflow:tensorflow-lite-gpu` - 2.6.0 -> 2.10.0
- `org.tensorflow:tensorflow-lite-gpu-api` -  -> 2.10.0
- `org.tensorflow:tensorflow-lite-support-api` -  -> 0.4.2
- `org.checkerframework:checkerqual` -  -> 3.25.0

### Describe your contribution

updated packages and dependencies

### CODE REVIEW

File: source/com.google.android.gms/play-services-tflite-gpu/Transforms/Metadata.Namespaces.xml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+        path="/api/package[@name='com.google.android.gms.tflite.internal.tflite_gpu']" 
+        name="managedName"
+        >
+        Android.Gms.TfLite.Internal.TfLiteGpu
+    </attr>
+    <attr 
+        path="/api/package[@name='com.google.android.gms.tflite.gpu']" 
+        name="managedName"
+        >
+        Android.Gms.TfLite.TfLite.Gpu
```
Comment (jpobst) [inline source/com.google.android.gms/play-services-tflite-gpu/Transforms/Metadata.Namespaces.xml:12]: Let's remove the duplicate `TfLite` here.

File: source/com.google.android.gms/play-services-tflite-gpu/Transforms/Metadata.Namespaces.xml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    </attr>
+    <attr 
+        path="/api/package[@name='com.google.android.gms.tflite.gpu.support']" 
+        name="managedName"
+        >
+        Android.Gms.TfLite.TfLite.Gpu.Support
```
Comment (jpobst) [inline source/com.google.android.gms/play-services-tflite-gpu/Transforms/Metadata.Namespaces.xml:18]: Same

File: source/com.google.android.gms/play-services-tflite-gpu/Transforms/Metadata.Namespaces.xml (hunk 1 above)
Comment (moljac) [reply source/com.google.android.gms/play-services-tflite-gpu/Transforms/Metadata.Namespaces.xml:12]: copypasta mistake

File: source/com.google.android.gms/play-services-tflite-gpu/Transforms/Metadata.Namespaces.xml (hunk 2 above)
Comment (moljac) [reply source/com.google.android.gms/play-services-tflite-gpu/Transforms/Metadata.Namespaces.xml:18]: copypasta mistake

**Review by jpobst — APPROVED** [review]

**Pull request merged**